clix/src/main/kotlin/so/clix/services/NotificationService.kt

```
@@ -225,28 +247,71 @@ internal class NotificationService(
+                ClixLogger.debug("Notification permission already granted")
+                return true
+            }
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                ClixLogger.debug("Requesting notification permission")
+                PermissionActivity.requestPermission(context)
+            } else {
+                ClixLogger.debug("Notification permission granted at install time (Android < 13)")
+                true
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)